Library/Homebrew/test/spec_helper.rb

```diff
@@ -179,6 +199,7 @@
   end
 
   config.around do |example|
+    undef find_files if defined? find_files
     def find_files
```

Comment:
This can be defined outside the block. Probable best to put it in a `Test::Helper` module.

Reply:
Great call, done